01-shopping-cart-service-scala/deployment/Akka-CRD-usercode.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
```

Comment:
This is a user-provided manifest to deploy an App with a cluster made of two replicasets (each with their roles and settings). It is meant to inherit multiple default values (See the other YAML on this PR).